### ISSUE TITLE
schedule function error for some older milb seasons

### ISSUE THREAD

**Issue by tonci-j** [tnestico/mlb_scraper#2]
I ran into the following error when trying to get the 2009 AAA schedule:

>>> schedule = scraper.get_schedule(year_input=[2009], sport_id=[11], game_type=['R'])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python312\Lib\site-packages\api_scraper.py", line 110, in get_schedule
    venue_id = [item for sublist in [[y['venue']['id'] for y in x['games']] for x in game_call['dates']] for item in sublist]
                                      ~~~~~~~~~~^^^^^^
KeyError: 'id'

It also happened to:
sport_id=[12] for 2006
sport_id=[14] for 2008, 2007, 2006

**Comment by tnestico** [comment]
Updated with a try/except condition

**Comment by tonci-j** [comment]
Now it doesn't pull any info for the error years:

>>> schedule = scraper.get_schedule(year_input=[2009], sport_id=[11], game_type=['R'])
No Data for Selected Parameters
>>> print(schedule)
None

I'm not sure that's helpful since the jsons for those sport_id do exist.

I poked around the previous version of the code and modded it, removing venue_id and venue_name from the schedule function since that was in the error message. It worked and allowed the remaining game_id, time, date, away, home, and state fields to come through (though for me, game_id was primary need). From there I was able to use the gathered game_id to get game_data as normal. It seems there's some error in the json for the venue_id and venue_name info for those particular year/sport_id combinations.
